### ISSUE TITLE
[Accessibility][Screen Readers]: No name is defined for remove button present next to selected colors. (Bug 3) 

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10822]
Breakout of issue #8655

**Comment by joschect** [comment]
Tag picker should be brought in line with the people picker, the x button shouldn't be a focusable tab stop since backspace works fine. Then there would be no need to have a name defined for the X. @betrue-final-final could you confirm that that fix is acceptable?

**Comment by xugao** [comment]
Here is people picker item:
https://github.com/OfficeDev/office-ui-fabric-react/blob/07b6b2fb036671a8d86d8efdf105f4a46be92079/packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItem.tsx#L38-L56

looks like the code intended to make X focusable as part of the sub focus zone, but it's not (so potentially that's a bug with people picker item, rather than intended behavior)

Going to wait for @betrue-final-final to confirm expected behavior before i make any fix
